unittests/allplatformstests.py

```diff
@@ -4633,7 +4633,10 @@ def test_symlinked_subproject(self):
         symlinked_subproject = os.path.join(testdir, 'subprojects', 'symlinked_subproject')
         if not os.path.exists(subproject_dir):
             os.mkdir(subproject_dir)
-        os.symlink(subproject, symlinked_subproject)
+        try:
+            os.symlink(subproject, symlinked_subproject)
+        except OSError:
+            raise SkipTest("Symlinks are not available on this machine")
```

Comment:
Hmm. How does this normally pass in CI?

Reply:
symlinks are possible on Windows, but require either elevated privileges or enabling developer mode.

Reply:
Makes sense, we do something similar in e.g. minstall...